AGENTS.md

```
@@ -560,6 +566,90 @@ Links embedded in transactional emails must use the correct dashboard host. Temp
+       fontFamily: FONTS.display,
+       basePx: 54,
+       lineHeight: 1.05,
+       allowWrap: true
+     }
+   } as const;
+   ```
+
+4. **CSS styles must use the dedicated layout variables**:
+   - Reference the same dedicated layout constants (e.g. paddings, gaps, column widths, border dimensions) in the template's CSS styles.
+   - Container max dimensions in CSS should reflect the total outer boundary where needed (e.g., inner width/height from `FIELDS` plus the padding):
+   ```css
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)